kingpin/actors/aws/test/test_ecs.py

```diff
@@ -24,77 +25,78 @@ def test_no_failures(self):
 
     @testing.gen_test
     def test_schema_ok_minimal(self):
-        self._test(
-            {
-                'family': '',
-                'containerDefinitions': [],
-            })
+        self._test({
+            'family': '',
+            'containerDefinitions': []})
 
     @testing.gen_test
     def test_schema_ok_complex(self):
-        self._test(
-            {
-                'family': 'family-name',
-                'containerDefinitions': [
-                    {
-                        'name': 'test',
-                        'image': 'test-image',
-                        'cpu': 1,
-                        'memory': 2,
-                        'links': ['a'],
-                        'portMappings': [{
-                            'containerPort': 8000,
-                            'hostPort': 80,
-                            'protocol': 'tcp'
-                        }],
-                        'essential': True,
-                        'environment': [
-                            {'name': 'var1', 'value': 'value1'},
-                            {'name': 'var2', 'value': 'value2'}
-                        ],
-                    },
-                    {
-                        'name': 'a',
-                        'image': 'a-image'
-                    }
-                ],
-                'volumes': [{
-                    'name': 'volume',
-                    'host': {
-                        'sourcePath': '/mountme'
-                    }
-                }]
-            })
+        self._test({
+            'family': 'family-name',
+            'containerDefinitions': [
+                {
+                    'name': 'test',
+                    'image': 'test-image',
+                    'cpu': 1,
+                    'memory': 2,
+                    'links': ['a'],
+                    'portMappings': [{
+                        'containerPort': 8000,
+                        'hostPort': 80,
+                        'protocol': 'tcp'
+                    }],
+                    'essential': True,
+                    'environment': [
+                        {'name': 'var1', 'value': 'value1'},
+                        {'name': 'var2', 'value': 'value2'}
+                    ],
+                },
+                {
+                    'name': 'a',
+                    'image': 'a-image'
+                }
+            ],
+            'volumes': [{
+                'name': 'volume',
+                'host': {
+                    'sourcePath': '/mountme'
+                }
+            }]
+        })
 
     @testing.gen_test
-    def test_schema_fail_extra(self):
-        with self.assertRaises(exceptions.InvalidOptions):
-            self._test(
-                {
-                    'other stuff': 0,
-                    'family': 'name',
-                    'containerDefinitions': [],
-                    'extra': 'read all about it'
-                })
+    def test_schema_ok_extra(self):
```

Comment:
I do love how complete you are here... but I am willing to trust that jsonschema is doing its job. :)